the 2 things to notice is 4 according to me (#1287)
elixir-lang/elixir-lang.github.com: getting-started/mix-otp/introduction-to-mix.markdown

end
```

It is important to note a couple of things:

1. the test file is an Elixir script file (`.exs`). This is convenient because we don't need to compile test files before running them;

2. we define a test module named `KVTest`, in which we [`use ExUnit.Case`](https://hexdocs.pm/ex_unit/ExUnit.Case.html) to inject the testing API;

> 3. we use one of the injected macros, `doctest` (TODO: for this I do not find documentation), to indicate that the `KV` module contains doctests (we are not going to discuss these further for the moment);

Reviewer: ```suggestion
3. we use one of the injected macros, [`doctest/1`](https://hexdocs.pm/ex_unit/ExUnit.DocTest.html#doctest/2), to indicate that the `KV` module contains doctests (we will discuss those in a later chapter);
```

> 4. we use the `text/2` macro to define a simple test;

Reviewer: ```suggestion
4. we use the [`test/2`](https://hexdocs.pm/ex_unit/ExUnit.Case.html#test/3) macro to define a simple test;
```
